includes/framework/QInstallationValidator.class.php

```diff
@@ -161,6 +161,34 @@ public static function Validate() {
 			$result[] = $obj;
 		}
 
+		if (!file_exists(__IMAGE_CACHE__)) {
```

Comment:
No, `_assets` version is to construct URLs. here we need full path in a filesystem with `__DOCROOT__` applied.